src/Mobs/Monster.cpp

```diff
@@ -257,7 +257,7 @@ void cMonster::MoveToWayPoint(cChunk & a_Chunk)
 	}
 
 	Vector3d Distance = m_NextWayPointPosition - GetPosition();
-	if ((Distance.x != 0) || (Distance.z != 0))
+	if ((Distance.x != 0.0f) || (Distance.z != 0.0f))
```

Comment:
Do these really trigger a warning? I thought they didn't, last time I checked. A zero is a zero in any number representation.